package.json

```diff
@@ -77,8 +77,8 @@
     "build:integration": "ng run integration:build:production",
     "build:integration:ssr": "yarn build:integration && ng run integration:server:production",
     "// - TESTING": "Test all modules #requires yarn build:packages",
-    "test": "ng test",
```

Comment:
Why are we moving away from `ng test`?

Reply:
Because `@angular-builders/jest` is not supported with Jest 26 and correctly Angular CLI 9, 
so you can just call the command:

```
$ jest --config jest.config.js 
```

Reply:
good catch